packages/specifications/components/form-field/specifications-form-field.mdx

```diff
@@ -0,0 +1,13 @@
+import {Description} from '@storybook/addon-docs';
+
+[//]: # (import Specs from '@ovhcloud/ods-core/src/components/form-field/docs/spec.md';)
```

Comment:
Here's why specification doc is not generated:
[//]: # (import Specs from '@ovhcloud/ods-core/src/components/form-field/docs/spec.md';) to `import Specs from '@ovhcloud/ods-core/src/components/form-field/docs/spec.md';`

Reply:
This is why! Thank you